packages/next/src/export/index.ts

```diff
@@ -422,6 +422,7 @@ export async function exportAppImpl(
       isAppPPREnabled: checkIsAppPPREnabled(nextConfig.experimental.ppr),
       clientTraceMetadata: nextConfig.experimental.clientTraceMetadata,
       swrDelta: nextConfig.experimental.swrDelta,
+      after: nextConfig.experimental.after === true,
```

Comment:
nit pick
```suggestion
      after: !!nextConfig.experimental.after,
```

Reply:
Changed this (and another occurence) to `?? false` -- i prefer not to `!!` out of vague paranoia that, if that attribute were to ever become something like `undefined | boolean | 'some-string'`, we'd accidentally collapse the string value into `true`, which'd end up being very confusing to debug 😅